src/sysdolphin/baselib/fobj.c

```diff
@@ -1,10 +1,7 @@
-#include "forward.h"
-
-#include "fobj.h"
```

Comment:
This pattern should work universally in common editors, and the Make toolchain is able to recognize it as well. `clang-format` is set up to sort includes that match the name of the current file above everything else. Were you having an issue with this format?

Reply:
I change the path of fobj.h to resolve a build error.

Forward wasn't needed by the file, so I removed it.